sources/api/prairiedog/src/main.rs

```diff
@@ -46,12 +55,22 @@ const KEXEC_CMD_LINE: &str = "maxcpus=1 systemd.unit='capture-kernel-dump.servic
 // Used to pass None to nix::mount::mount
 const NONE: Option<&'static [u8]> = None;
 
+// Boot config/initrd related consts
+const BOOTCONFIG_INITRD_PATH: &str = "/var/lib/bottlerocket/bootconfig.data";
+const PROC_BOOTCONFIG: &str = "/proc/bootconfig";
+const DEFAULT_BOOTCONFIG_STR: &str = r#"kernel = ""
+init = ""
+"#;
```

Comment:
nit: maybe this for readability?
```suggestion
const DEFAULT_BOOTCONFIG_STR: &str = r#"
kernel = ""
init = ""
"#;
```